tests/epics/signal/test_signals.py

```diff
@@ -901,3 +904,21 @@ async def test_signals_created_for_not_prec_0_float_cannot_use_int(ioc: IOC):
         TypeError, match=f"{ioc.protocol}:float_prec_1 has type float not int"
     ):
         await sig.connect()
+
+
+async def test_can_read_using_ophyd_async_then_ophyd(ioc: IOC):
+    oa_read = f"{ioc.protocol}://{PV_PREFIX}:{ioc.protocol}:float_prec_1"
+    ophyd_read = f"{PV_PREFIX}:{ioc.protocol}:float_prec_0"
+
+    ophyd_async_sig = epics_signal_rw(float, oa_read)
+    await ophyd_async_sig.connect()
+    ophyd_signal = EpicsSignal(ophyd_read)
+    ophyd_signal.wait_for_connection()
```

Comment:
Can we put any timeouts in here, or does it just hang anyway?

Reply:
It hangs either way but I've added it anyway